sys/net/network_layer/ng_sixlowpan/frag/rbuf.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_ng_sixlowpan_frag
+ * @{
+ *
+ * @file
+ * @internal
+ * @brief   6LoWPAN reassembly buffer
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef NG_SIXLOWPAN_FRAG_RBUF_H_
+#define NG_SIXLOWPAN_FRAG_RBUF_H_
+
+#include <inttypes.h>
+
+#include "net/ng_pkt.h"
+#include "timex.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define RBUF_L2ADDR_MAX_LEN (8U)    /**< maximum length for link-layer addresses */
+#define RBUF_SIZE           (4U)    /**< size of the reassembly buffer */
+#define RBUF_TIMEOUT        { 3, 0 }/**< timeout for reassembly as timex */
```

Comment:
I know what a timex value is, but may you can specify that these are seconds anyway.